src/components/Header/Header.jsx

```diff
@@ -25,7 +25,7 @@ export const Header = (isOpen) => {
       <Flex
         align="center"
         justify="space-between"
-        maxWidth="1200px"
+        maxW="container.xl"
```

Comment:
This prop set max-width of the header to '1280px', to keep our max-width consistent